cfme/control/explorer/conditions.py

```diff
@@ -73,7 +74,8 @@ class ConditionsAllView(ControlExplorerView):
     def is_displayed(self):
         return (
             self.in_control_explorer and
-            self.title.text == "All Conditions" and
+            # there is a BZ 1683697 that some Condition view is shown for All Conditions
```

Comment:
:+1: was literally making a comment about the duplicate, thanks.

Reply:
I've found them while I was looking for duplicates.